src/main/scala/dev/kamu/cli/commands/AddCommand.scala

```diff
@@ -5,20 +5,20 @@ import dev.kamu.cli.{
   MetadataRepository,
   MissingReferenceException
 }
-import org.apache.hadoop.fs.{FileSystem, Path}
+import org.apache.hadoop.fs.FileSystem
 import org.apache.log4j.LogManager
 
 class AddCommand(
   fileSystem: FileSystem,
   metadataRepository: MetadataRepository,
-  manifests: Seq[Path]
+  manifests: Seq[String]
```

Comment:
should we use the `URI` type here, that's what I'm using throughout the code to represent some resource (local or on the web)